src/main/deploy/pathplanner/FullAuto.path

```diff
@@ -99,17 +24,17 @@
     },
     {
       "anchorPoint": {
-        "x": 4.55902689583157,
```

Comment:
We should get in the habit or setup a process for documenting the autonomous paths so programming and rive can keep them all straight.

Reply:
Yes, we should remember to add that once we implement an auto/path chooser.